hack/quick-install.sh

```diff
@@ -3,6 +3,8 @@
 set -o errexit
 set -o xtrace
 
+bash hack/install-database.sh
```

Comment:
Also in this file, we need to add the copyright paragraph.

Reply:
Added